### ISSUE TITLE
Update offer should expose more information

### ISSUE THREAD

**Issue by Dilshat** [subutai-io/peer-os#1814]
Currently SS shows only that an update is available. We need to expose information describing the update to user.

**Comment by Dilshat** [comment]
Blocked by https://github.com/optdyn/hub/issues/2997

**Comment by Dilshat** [comment]
Closed as duplicate of https://github.com/subutai-io/peer-os/issues/2098
